Solutions/MSTelemetrySDK.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28307.645
+# Visual Studio Version 16

```

Comment:
Sorry, I violated my own principle of not committing Visual Studio 2019 files. If it opens in Visual Studio 2017, then we are still good. I can revert it, if you folks insist on reverting it.